tests/cypress/integration/actions_users/issue_1810_login_logout.js

```diff
@@ -97,5 +130,25 @@ context('When clicking on the Logout button, get the user session closed.', () =
             cy.url().should('include', '/auth/login');
             cy.closeNotification('.cvat-notification-notice-login-failed');
         });
+
+        it('Empty user and empty password', () => {
+            loginWithoutCredentials(false, false);
+        });
+
+        it('Correct user and empty password', () => {
+            loginWithoutCredentials(Cypress.env('user'), false);
+        });
+
+        it('Empty user and correct password', () => {
+            loginWithoutCredentials(false, Cypress.env('password'));
+        });
+
+        it('Incorrect user and empty password', () => {
+            loginWithoutCredentials('randomUser123', false);
+        });
+
+        it('Empty user and incorrect password', () => {
+            loginWithoutCredentials(false, 'randomPassword123');
+        });
```

Comment:
Can we parametrize call of ``loginWithoutCredentials`` in a ``for`` loop? Like 
```
for (const [username, password] of [[user1, pass1], [user2, pass2] ,...) { 
    loginWithoutCredentials(username, password)
}
```

I personally do not think we need such number of negative tests, it is like we are trying to test the framework we are using on UI

Reply:
I parameterized this test and the test with the wrong password or name in the way you suggested.

This is the entry point, all users go through it, I think it is reasonable to check all corner cases. This test takes seconds to pass.